doc/manual.rst

```diff
@@ -8,8 +8,7 @@ Nim Manual
 .. contents::
 
 
-  "Complexity" seems to be a lot like "energy": you can transfer it from the end
-  user to one/some of the other players, but the total amount seems to remain
+  "Complexity" seems to be a lot like "energy": you can transfer it from the end-user to one/some of the other players, but the total amount seems to remain
```

Comment:
There's another long line here and other instances in manual.rst, nimc.rst, and tut1.rst.

Reply:
fixed :)